### ISSUE TITLE
volume from doesn't appear to work

### ISSUE THREAD

**Issue by taylodl** [rhuss/docker-maven-plugin#73]
I start a maven container and give it the name 'maven' using docker's --name option. In my pom.xml I've configured my image as follows:

```xml
<image>
   <name>taylodl/wls-mydomain</name>
  <run>
   <volumes>
      <from>maven</from>
   </volumes>
         .
         .
         .  
```
When I run maven I get the following error: 

```
[ERROR] Failed to execute goal org.jolokia:docker-maven-plugin:0.10.6-SNAPSHOT:start (start) on project jaxRS-maven: No container start order could be found: Cannot resolve image dependencies for start order
[ERROR] Unresolved images:
[ERROR] taylodl/wls-mydomain depends on maven
```

If I run taylodl/wls-mydomain using the docker run option "--volumes-from maven" then everything runs as I expect and the volumes are mounted where I expect. I was expecting <volumes><from> to accomplish the same thing.

**Comment by jgangemi** [comment]
that functionality doesn't currently exist but i think that could be easily added. an additional element (maybe `external`) could be added to the existing `volumes` element that could be used to reference volumes created outside the container.

**Comment by jgangemi** [comment]
fixed in 0.13.2
